portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Icon.path

```diff
@@ -373,6 +373,11 @@
 	<td>//*[contains(.,'${key_text}')]//*[*[name()='svg'][contains(@class,'icon-ellipsis')]]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>SPECIFIC_ELLIPSIS</td>
+	<td>//*[contains(@class, 'text-truncate') and contains(text(), '${key_text}')]//..//..//..//*[contains(.,'')]//*[*[name()='svg'][contains(@class,'icon-ellipsis')]]</td>
+	<td></td>
+</tr>
```

Comment:
I think this xpath could be smaller. Would you mind to show me how you retrieved this xpath?

Reply:
Yes, of course.